# Systematic Literature Review

A Systematic Literature Review (SLR) is well defined and structured method or process for identifying, selecting and critically appraising literature on a specific topic.

7 steps are considered to conduct a SLR:

1. Define the resarch question
2. Define the scope, objectives and inclusion or exclusion criteria.
3. Search for studies in relevant databases.
4. Screen and select studies for detailed review using the scope, objectives, and criteria defined in Step 2.
5. Extract data from studies resulting from step 4.
6. Synthesize results.
7. Disseminate results.

This python Notebook steps 1 to 4 serves as reliable proof and as an example of the SLR conducted by Santiago García Gil during his PhD.

## Step 1: Define the research question

To define the research question the PICOC framework is used:
|     PICOC    	|  Definition                                                                                                          	|
|:------------:	|--------------------------------------------------------------------------------------------------------------------	|
| Population   	| Mesh/Ad Hoc networks where Unmanned Aerial Systems (UAS) are used as the platform of nodes.                        	|
| Intervention 	| Application of Deep Reinforcement Learning (DRL)                                                                   	|
| Comparison   	| No DRL or application of traditional methods.                                                                      	|
| Outcome      	| Impact in network-level metrics such as throughput, latency, jitter, packet loss, delivery ratio, and reliability. 	|
| Context      	| Computer networks                                                                                                  	|

## Step 2: Define the scope, objectives and inclusion or exclusion criteria
The thesis of Santiago García Gil is deeply related to 3 topics: ad hoc computer networks, UAS and DRL. Given these facts and the result of step 1, the scope of the SLR is defined as follows:

- Scope:
    - UAS platforms acting as nodes, relays, or routers in ad hoc network topologies.
    - Applications of Deep Reinforcement Learning for network control, routing, resource allocation, connectivity management, and QoS improvement.
    - Evaluation focused on network-level metrics such as throughput, latency, jitter, packet loss, delivery ratio, and reliability.
    - Context limited to computer networking research rather than UAV flight dynamics or perception.

- Objectives:
    1. Identify and categorize DRL methods applied to UAS-enabled ad hoc network problems.
    2. Study the usefulness given the reported performance improvements and QoS results across comparable approaches.

- Inclusion criteria (IC):
    1. (IC1) Peer-reviewed journal or conference papers written in English.
    2. (IC2) Studies published from 2015 to 2026 that apply DRL in aerial ad hoc networking scenarios.
    3. (IC3) Papers with experimental or simulation results on network performance or QoS.
    4. (IC4) Research with a clear network-level focus, not limited to UAV path planning, vision, or flight control.

- Exclusion criteria (EC):
    - (EC1) Works that use classical RL without deep learning components.
    - (EC2) UAV research focused only on navigation, mapping, or sensing without ad hoc network operations.
    - (EC3) Ad hoc networking studies without multihop aerial/UAV nodes.
    - (EC4) Survey papers, editorials, patents, or non-peer-reviewed reports without original results.
    - (EC5) Studies that do not report measurable network metrics or QoS outcomes.

- Screening:
    1. Data reconciliation from resulting articles from different academical databases (elimination of: duplicates, non journal or conference papers and/or retracted papers).
    2. Title/abstract screening. Artificial Intelligence (AI) assisted with human in the loop. Inclusion/exclusion criteria defined in Step 2 is used.
    3. Full-text screening of remaining papers to confirm DRL use, network focus, and reported QoS/network metrics.



## Step 3: Seach for studies in relevant databases
Well known computer and telecomunication databases are considered for the current SLR, namely: IEEE Xplore, Scopus (Elsevier) and ACM Digital Library.

The following query has been designed for use in the advance search functionality of [IEEE Xplore](https://ieeexplore.ieee.org/search/advanced/command):
```plaintext
("All Metadata":"mesh" OR "All Metadata":"ad hoc" OR "All Metadata":"vanet" OR "All Metadata":"fanet") 
    AND ("All Metadata":"network" OR "All Metadata":"networks" OR "All Metadata":"networking") 
    AND ("All Metadata":"drl" OR "All Metadata":"deep reinforcement learning" OR "All Metadata":"reinforcement learning")
    AND ("All Metadata":"uav" OR "All Metadata":"uas" OR "All Metadata":"drone" OR "All Metadata":"drones" ) 
    AND ("All Metadata":"latency" OR "All Metadata":"jitter" OR "All Metadata":"qos" OR "All Metadata":"quality of service" OR "All Metadata":"throughtput")
    AND NOT "Publication Title":"review" 
    AND NOT "Publication Title":"survey" 
```

Similarly, the following query is designed for [Scopus](https://www.scopus.com/search/form.uri?display=advanced):
```plaintext
(TITLE-ABS-KEY("mesh") OR TITLE-ABS-KEY("ad hoc") OR TITLE-ABS-KEY("vanet") OR TITLE-ABS-KEY("fanet")) 
    AND (TITLE-ABS-KEY("network") OR TITLE-ABS-KEY("networks") OR TITLE-ABS-KEY("networking")) 
    AND (TITLE-ABS-KEY("drl") OR TITLE-ABS-KEY("deep reinforcement learning") OR TITLE-ABS-KEY("reinforcement learning"))
    AND (TITLE-ABS-KEY("uav") OR TITLE-ABS-KEY("uas") OR TITLE-ABS-KEY("drone") OR TITLE-ABS-KEY("drones") ) 
    AND (TITLE-ABS-KEY("latency") OR TITLE-ABS-KEY("jitter") OR TITLE-ABS-KEY("qos") OR TITLE-ABS-KEY("quality of service") OR TITLE-ABS-KEY("throughtput"))
    AND NOT TITLE("review") 
    AND NOT TITLE("survey") 
```

As of 2026/05/15, the query yields 65 entries.

Finally, for [ACM Digital Library](https://dl.acm.org/search/advanced) the query is as follows:
```plaintext
AllField:(mesh OR "ad hoc" OR vanet OR fanet) 
    AND AllField:(network OR networks OR networking) 
    AND AllField:(drl OR "deep reinforcement learning" OR "reinforcement learning") 
    AND AllField:(uav OR uas OR drone OR drones) 
    AND AllField:(latency OR jitter OR qos OR "quality of service" OR "throughtput") 
    AND Title:(NOT survey) 
    AND Title:(NOT review) 
    AND Abstract:(NOT survey) 
    AND Abstract:(NOT review)
```
For ACM Digital Library, only results from 2018 onwards are considered.

As of 2026/05/15, queries yields 76, 65 and 938 entries, respectively.

## Step 4: Screen and select studies for detailed review using the scope, objectives, and criteria defined in Step 2.

The following staged screening process is going to be followed:
1. Pruning and duplication removal.
2. Title + Abstract screening.
3. Full text screening.

### Step 4.1: Pruning and duplication removal

In [ ]:
import bibtexparser

with open('step_3_results/acm.bib') as bibtex_file:
   acm_database = bibtexparser.load(bibtex_file)
with open('step_3_results/ieee.bib') as bibtex_file:
   ieee_database = bibtexparser.load(bibtex_file)
with open('step_3_results/scopus.bib') as bibtex_file:
   scopus_database = bibtexparser.load(bibtex_file)   

In [3]:
import pandas as pd

acm_df = pd.DataFrame(acm_database.entries)
ieee_df = pd.DataFrame(ieee_database.entries)
scopus_df = pd.DataFrame(scopus_database.entries)

print('Information about ACM Library query result')
print(f'No. of entries: {len(acm_df)}')
print(f'Columns: {len(acm_df.columns)}: {acm_df.columns.tolist()}')
print(f'Non-null values: {acm_df[['title', 'keywords', 'abstract']].count()}')
print()
print('Information about IEEE Xplore query result')
print(f'No. of entries: {len(ieee_df)}')
print(f'Columns: {len(ieee_df.columns)}: {ieee_df.columns.tolist()}')
print(f'Non-null values: {ieee_df[['title', 'keywords', 'abstract']].count()}')
print()
print('Information about Scopus query result')
print(f'No. of entries: {len(scopus_df)}')
print(f'Columns: {len(scopus_df.columns)}: {scopus_df.columns.tolist()}')
print(f'Non-null values: {scopus_df[['title', 'keywords', 'abstract']].count()}')
print()

Information about ACM Library query result
No. of entries: 938
Columns: 27: ['series', 'location', 'keywords', 'numpages', 'pages', 'booktitle', 'abstract', 'doi', 'url', 'address', 'publisher', 'isbn', 'year', 'title', 'author', 'ENTRYTYPE', 'ID', 'articleno', 'month', 'journal', 'number', 'volume', 'issue_date', 'issn', 'note', 'editor', 'edition']
Non-null values: title       938
keywords    102
abstract    242
dtype: int64

Information about IEEE Xplore query result
No. of entries: 76
Columns: 15: ['month', 'issn', 'doi', 'keywords', 'abstract', 'pages', 'number', 'volume', 'year', 'title', 'journal', 'author', 'ENTRYTYPE', 'ID', 'booktitle']
Non-null values: title       76
keywords    76
abstract    76
dtype: int64

Information about Scopus query result
No. of entries: 65
Columns: 18: ['note', 'source', 'publication_stage', 'type', 'keywords', 'author_keywords', 'abstract', 'url', 'doi', 'pages', 'number', 'volume', 'journal', 'year', 'title', 'author', 'ENTRYTYPE', 'ID']
Non-null

In [4]:
# Show a sample of entries with keywords and abstract

acm_df_filtered = acm_df[acm_df['abstract'].isnull() & acm_df['keywords'].isnull()]
print(f'No. of entries without keywords and abstract in ACM Library query result: {acm_df_filtered.shape[0]}')
print(acm_df_filtered['title'].sample(30))
print('-'*80)

# It seems that most of the entries in the ACM Library that do not have keywords nor abstract are proceedings of conferences.

acm_df_proceedings = acm_df[acm_df['title'].str.contains('Proceeding', case=False, na=False)]
print(f'No. of entries with "Proceeding" in the title in ACM Library query result: {acm_df_proceedings.shape[0]}')
print(acm_df_proceedings['title'].sample(30))
print('-'*80)

# Check the not intersecting entries
acm_df_filtered = pd.concat([acm_df_filtered, acm_df_proceedings], axis=0).drop_duplicates(keep=False)
print(f'No. of entries with keywords and abstract in ACM Library query result that are also proceedings of conferences: {acm_df_filtered.shape[0]}')
print(acm_df_filtered['title'].sample(30))
print('-'*80)

# It turns of that non intersecting entries are also proceedings of conferences that have abstracts.
# Remove all entries with "Proceeding" in the title from the ACM Library query result
acm_df_filtered = acm_df[~acm_df['title'].str.contains('Proceeding', case=False, na=False)]
print(f'No. of entries with keywords and abstract in ACM Library query result after removing proceedings of conferences: {acm_df_filtered.shape[0]}')
print(acm_df_filtered[['title', 'abstract', 'keywords']].count())

# Some nulls in the abstract column, but they are not proceedings of conferences. Let's check them.
acm_df_aux = acm_df_filtered[acm_df_filtered['abstract'].isnull() & acm_df_filtered['keywords'].isnull()]

print(acm_df_aux[['title', 'abstract', 'keywords']].sample(20))

# They seem to be entries of posters, demos and not useful for our SLR. Let's remove them.
acm_df_filtered = acm_df_filtered[~(acm_df_filtered['abstract'].isnull() & acm_df_filtered['keywords'].isnull())]
print(f'No. of entries with keywords and abstract in ACM Library query result after removing entries without keywords and abstract: {acm_df_filtered.shape[0]}')
print(acm_df_filtered[['title', 'abstract', 'keywords']].count())

acm_df_aux = acm_df_filtered[acm_df_filtered['abstract'].isnull()]
print(acm_df_aux['title'])
# They are unrelated old papers that are not relevant for our SLR. Let's remove them.
acm_df_filtered = acm_df_filtered[~acm_df_filtered.title.isin(acm_df_aux['title'])]
print(f'No. of clean entries in ACM Library query result: {acm_df_filtered.shape[0]}')
print(acm_df_filtered[['title', 'abstract', 'keywords']].count())


No. of entries without keywords and abstract in ACM Library query result: 696
309    ICIEAI '25: Proceedings of the 2025 3rd Intern...
579    DSAI '25: Proceedings of the 2025 2nd Internat...
483    GoodIT '23: Proceedings of the 2023 ACM Confer...
855    HSCC '23: Proceedings of the 26th ACM Internat...
618    MemSys '25: Proceedings of the International S...
573    CIBDA '24: Proceedings of the 5th Internationa...
199    CACML '23: Proceedings of the 2023 2nd Asia Co...
744    OzCHI '24: Proceedings of the 36th Australasia...
585    ICBDT '23: Proceedings of the 2023 6th Interna...
656    UIST Adjunct '24: Adjunct Proceedings of the 3...
546    MIG '24: Proceedings of the 17th ACM SIGGRAPH ...
720    ASSETS '24: Proceedings of the 26th Internatio...
829    ICMI '22: Proceedings of the 2022 Internationa...
266    IOT '25: Proceedings of the 15th International...
685    CPS-IoT Week '23: Proceedings of Cyber-Physica...
558    FOGA '25: Proceedings of the 18th ACM/SIGEVO C...
171    ADI

In [5]:
acm_df_aux = acm_df_filtered[acm_df_filtered['abstract'].isnull() | acm_df_filtered['keywords'].isnull()][['title', 'abstract', 'keywords']]
print(acm_df_aux.count())
print(acm_df_aux[acm_df_aux['abstract'].isnull()])

title       47
abstract    47
keywords     0
dtype: int64
Empty DataFrame
Columns: [title, abstract, keywords]
Index: []


In [6]:
# Let's check if there are proceedings of conferences in the IEEE Xplore query result
ieee_df_proceedings = ieee_df[ieee_df['title'].str.contains('Proceedings', case=False, na=False)]
print(f'No. of entries with "Proceedings" in the title in IEEE Xplore query result: {ieee_df_proceedings.shape[0]}')
# It is already clean


No. of entries with "Proceedings" in the title in IEEE Xplore query result: 0


In [7]:
# Same for Scopus, check entries with missing keywords and check if there are proceedings of conferences in the Scopus query result
scopus_df_filtered = scopus_df[scopus_df['keywords'].isnull()]
print(f'No. of entries without keywords and abstract in Scopus query result: {scopus_df_filtered.shape[0]}')
print(scopus_df_filtered['title'].head())
print('-'*80)
scopus_df_proceedings = scopus_df[scopus_df['title'].str.contains('Proceedings', case=False, na=False)]
print(f'No. of entries with "Proceedings" in the title in Scopus query result: {scopus_df_proceedings.shape[0]}')
print(scopus_df_proceedings['title'].head())
print('-'*80)
# There are some proceedings, so we will remove all entries with "Proceedings" in the title from the Scopus query result
scopus_df_filtered = scopus_df[~scopus_df['title'].str.contains('Proceedings', case=False, na=False)]
print(f'No. of entries in Scopus query result after removing proceedings of conferences: {scopus_df_filtered.shape[0]}')


No. of entries without keywords and abstract in Scopus query result: 7
12    A novel Q-learning-based secure routing scheme...
24    Multi-agent Deep Learning for QoS-aware Task S...
31    Delay aware downlink resource allocation schem...
36    AI-Driven UAV-Assisted Edge Computing for Rapi...
40    DIVANet 2018 - Proceedings of the 8th ACM symp...
Name: title, dtype: str
--------------------------------------------------------------------------------
No. of entries with "Proceedings" in the title in Scopus query result: 2
40    DIVANet 2018 - Proceedings of the 8th ACM symp...
44    Proceedings - 2020 IEEE 17th International Con...
Name: title, dtype: str
--------------------------------------------------------------------------------
No. of entries in Scopus query result after removing proceedings of conferences: 63


In [8]:
# Reconciliation of the three datasets
slr_df = pd.concat(
    [acm_df_filtered[['title', 'keywords', 'abstract']],
     ieee_df[['title', 'keywords', 'abstract']], 
     scopus_df_filtered[['title', 'keywords', 'abstract']]],
    ignore_index=True)

print('Information about the SLR DataFrame')
print(slr_df.count())
print(slr_df.columns)
# Lets check if there are duplicated entries in the SLR DataFrame
slr_df_duplicates = slr_df[slr_df.duplicated(subset=['title'], keep=False)]
print(f'No. of duplicated entries in the SLR DataFrame: {slr_df_duplicates.shape[0]}')

slr_df_clean = slr_df.drop_duplicates(subset=['title'], keep='first')
print(f'No. of clean entries in the SLR DataFrame (after removing duplicates): {slr_df_clean.shape[0]}')

# Lets sample some entries from the SLR DataFrame
print(slr_df_clean[['title', 'abstract', 'keywords']].sample(5))


Information about the SLR DataFrame
title       288
keywords    236
abstract    288
dtype: int64
Index(['title', 'keywords', 'abstract'], dtype='str')
No. of duplicated entries in the SLR DataFrame: 36
No. of clean entries in the SLR DataFrame (after removing duplicates): 270
                                                 title  \
106  CORE-lens: simultaneous communication and obje...   
128  Switch transformers: scaling to trillion param...   
126  Machine Learning with Confidential Computing: ...   
277  Multi-Agent Deep Reinforcement Learning in Fly...   
119  Quartz: time-as-a-service for coordination in ...   

                                              abstract  \
106  Optical camera communication (OCC) enabled by ...   
128  In deep learning, models typically reuse the s...   
126  Privacy and security challenges in Machine Lea...   
277  In the near future, 6G networks will be able t...   
119  Geo-distributed systems ranging from databases...   

                         

In [9]:
# Store the list to a CSV file
slr_df_clean.to_csv('slr_data.csv', index=True)

# There are 270 entries in the CSV file. 
# Manual check to remove retracted papers, and/or entries that are not relevant for the SLR and preprunning was not able to remove them.
# Indeces to be removed: 4 (retracted), 108 (proceedings), 121 (proceedings), 124 (proceedings), 135 (proceedings), 137 (proceedings), 146 (proceedings)
slr_df_clean = slr_df_clean.drop(index=[4, 108, 121, 124, 135, 137, 146])
print(f'No. of clean entries in the SLR DataFrame after manual check: {slr_df_clean.shape[0]}')

# Overwrite the CSV file with the clean entries
slr_df_clean.reset_index(drop=True, inplace=True)
slr_df_clean.to_csv('slr_data.csv', index=True)


No. of clean entries in the SLR DataFrame after manual check: 263


### Step 4.2: Title + abstract screening

| 0   	| title                                                                                                                                                                       	| IC1 	| IC2 	| IC3 	| IC4 	| EC1 	| EC2 	| EC3 	| EC4 	| EC5 	|
|-----	|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------	|-----	|-----	|-----	|-----	|-----	|-----	|-----	|-----	|-----	|
| 1   	| Beyond Static Populations: Efficient Delay-Constrained Scheduling for Dynamic Users via Deep Reinforcement Learning                                                         	| ✔️   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❔   	| ✔️   	| ✔️   	| ✔️   	|
| 2   	| **Topology Maintenance Optimization Algorithm Based on Deep Reinforcement Learning in High Dynamic Flying Ad-Hoc Networks**                                                 	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 3   	| A multi-Agent Deep Reinforcement Learning -based Task Offloading Strategy for LEO-MEC-Assisted UAV Systems                                                                  	| ✔️   	| ❌   	| ❌   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 4   	| Eagle: End-to-end Deep Reinforcement Learning based Autonomous Control of PTZ Cameras                                                                                       	| ✔️   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 5   	| A many-core accelerator design for on-chip deep reinforcement learning                                                                                                      	| ✔️   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 6   	| **Distributed trained Q-learning based AODV routing protocol for UAV ad hoc networks**                                                                                      	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 7   	| Enhancing Disaster Resilience with UAV-Assisted Edge Computing: A Reinforcement Learning Approach to Managing Heterogeneous Edge Devices                                    	| ✔️   	| ✔️   	| ❌   	| ❌   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	|
| 8   	| A general learning framework for open ad hoc teamwork Using graph-based policy learning                                                                                     	| ✔️   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 9   	| Optimizing Effectiveness and Defense of Drone Surveillance Missions via Honey Drones                                                                                        	| ✔️   	| ✔️   	| ❌   	| ❌   	| ✔️   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 10  	| Enhancing UAV Swarm Routing with Multi-Agent Attention Reinforcement Learning                                                                                               	| ✔️   	| ✔️   	| ❔❌  	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ❔❌  	|
| 11  	| **Deep Distributional Reinforcement Learning-Based Adaptive Routing With Guaranteed Delay Bounds**                                                                          	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 12  	| D3QN-Based Multi-Hop Offloading in Vehicular Networks                                                                                                                       	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ❌   	| ❌   	| ✔️   	| ✔️   	|
| 13  	| Multi-Agent Reinforcement Learning-Based Trading Decision-Making in Platooning-Assisted Vehicular Networks                                                                  	| ✔️   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 14  	| MADRL Based Joint Resource Allocation and 3D CoordinateOptimization of UAVs in Air-Ground Integrated IoT Networks                                                           	| ✔️   	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	|
| 15  	| Achieving Resilient and Self-Adaptive Topology Configuration in 3D UAV Networks                                                                                             	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ❔❌  	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 16  	| Online Inverse Reinforcement Learning Under Occlusion                                                                                                                       	| ✔️   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 17  	| PURE: A Predictive Path based UAV-IoT Network for Response oriented Energy-efficient Border Monitoring                                                                      	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ✔️   	| ❌   	|
| 18  	| LADAR: Low-Latency Distributed Articulation Point Detection and Mobility Recovery for FANETs                                                                                	| ✔️   	| ❔❌  	| ✔️   	| ✔️   	| ❔❌  	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 19  	| Maximizing Sampling Frequency for UAV Clusters in Spectrum Reconnaissance Applications                                                                                      	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 20  	| Hierarchical Cooperative Multi-Agent Reinforcement Learning with Skill Discovery                                                                                            	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 21  	| Intelligent Drone Surveillance with Efficient Charging Mechanism Using Eternal Vertex Cover and p-Dispersion                                                                	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 22  	| Beyond Convolutions YOLOv8 with DRFF-Based Channel-Mixing for Enhanced Drone Detection                                                                                      	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 23  	| Communication-Topology-preserving Motion Planning: Enabling Static Routing in UAV Networks                                                                                  	| ✔️   	| ❔❌  	| ✔️   	| ✔️   	| ❔❌  	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 24  	| E2P: An Energy Efficient Protocol for Human-Centric UAV-Based Autonomous Systems                                                                                            	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 25  	| Network Slicing-Assisted Task Transmission and Offloading Based on H-NOMA UAV Communication                                                                                 	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 26  	| VoI-based Situation-Aware Routing Protocol for Non-linear Underwater Communication Networks                                                                                 	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ❌   	| ❌   	| ✔️   	| ✔️   	|
| 27  	| Robust Permissioned Blockchain Consensus for Unstable Communication in FANET                                                                                                	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 28  	| Performance Evaluation of Network Slicing in Multi-Domain, Multi-Technology Wireless Networks                                                                               	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ❌   	| ❌   	| ✔️   	| ✔️   	|
| 29  	| Inverse reinforcement learning via nonparametric spatio-temporal subgoal modeling                                                                                           	| ✔️   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 30  	| A Resilient Control Strategy for Train-to-Train Communications under Jamming Attacks                                                                                        	| ✔️   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 31  	| Graph Transformer Aided Resource Virtualization Embedding in UAV Swarm Networks                                                                                             	| ✔️   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 32  	| Robust machine learning-enabled routing for highly mobile vehicular networks with PARRoT in ns-3                                                                            	| ✔️   	| ❔❌  	| ❔✔️  	| ✔️   	| ❔❌  	| ✔️   	| ✔️   	| ✔️   	| ❔❌  	|
| 33  	| **Learning Driven Mobility Control of Airborne Base Stations in Emergency Networks**                                                                                        	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 34  	| SDN in the stratosphere: loon's aerospace mesh network                                                                                                                      	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ❌   	| ❌   	| ✔️   	| ✔️   	|
| 35  	| kaNSaaS: Combining Deep Learning and Optimization for Practical Overbooking of Network Slices                                                                               	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 36  	| A Secure and Intelligent Data Sharing Scheme for UAV-Assisted Disaster Rescue                                                                                               	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 37  	| Sustainable Provision of URLLC Services for V2N: Analysis and Optimal Configuration                                                                                         	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 38  	| Sim-to-Real Transfer for Estimation over Wireless Networks                                                                                                                  	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 39  	| Optimizing Wireless Mesh Network Gateway Deployment Using an Enhanced Sparrow Optimization Algorithm                                                                        	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ❌   	| ❌   	| ✔️   	| ✔️   	|
| 40  	| EXPLORA: AI/ML EXPLainability for the Open RAN                                                                                                                              	| ✔️   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 41  	| Dynamic Computation Off-loading and Control based on Occlusion Detection in Drone Video Analytics                                                                           	| ✔️   	| ❌   	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ✔️   	|
| 42  	| Research on Data Collection Mechanism of Wireless Sensor Network Based on UAV                                                                                               	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	|
| 43  	| Resource Allocation for Stable LLM Training in Mobile Edge Computing                                                                                                        	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 44  	| Distributed Computation Offloading and Power Control for UAV-Enabled Internet of Medical Things                                                                             	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ❌   	| ❌   	| ✔️   	| ✔️   	|
| 45  	| Online incentive mechanism for task offloading with privacy-preserving in UAV-assisted mobile edge computing                                                                	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 46  	| Non-position-based UAV trajectory optimization for coverage maximization                                                                                                    	| ✔️   	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ❌   	|
| 47  	| Blockchain and Multi-Agent Learning Empowered Incentive IRS Resource Scheduling for Intelligent Reconfigurable Networks                                                     	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 48  	| Multi-Objective Optimization Approaches for Physical Layer Secure Communications Based on Collaborative Beamforming in UAV Networks                                         	| ✔️   	| ❌   	| ❌   	| ✔️   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 49  	| Collaborative Scheduling of Time-dependent UAVs, Vehicles and Workers for Crowdsensing in Disaster Response                                                                 	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 50  	| Maximizing Energy Efficiency of Period-Area Coverage With a UAV for Wireless Rechargeable Sensor Networks                                                                   	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 51  	| Optimizing Sectorized Wireless Networks: Model, Analysis, and Algorithm                                                                                                     	| ✔️   	| ❌   	| ❌   	| ✔️   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 52  	| Cookie-Jar: An Adaptive Re-configurable Framework for Wireless Network Infrastructures                                                                                      	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ❌   	| ❌   	| ✔️   	| ✔️   	|
| 53  	| Practical Efficient Microservice Autoscaling with QoS Assurance                                                                                                             	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 54  	| Resource Allocation and Secure Wireless Communication in the Large Model based Mobile Edge Computing System                                                                 	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 55  	| Joint Optimization of Task Offloading Content Caching and Resource Allocation in Vehicular Edge Computing                                                                   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ❌   	| ❌   	| ✔️   	| ✔️   	|
| 56  	| MAMS: Mobility-Aware Multipath Scheduler for MPQUIC                                                                                                                         	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ❌   	| ❌   	| ✔️   	| ✔️   	|
| 57  	| Learning to Schedule Network Resources Throughput and Delay Optimally Using Q&lt;sup&gt;+&lt;/sup&gt;-Learning                                                              	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ❌   	| ❌   	| ✔️   	| ✔️   	|
| 58  	| Distributed AI-Enabled Intelligent Control System for Enterprise Procurement and Supply-Chain Management                                                                    	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 59  	| Open Trends On TCP Performance Over Urban 5G mmWave Networks                                                                                                                	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	|
| 60  	| Attack-Resilient Connectivity Game for UAV Networks using Generative Adversarial Learning                                                                                   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ❌   	| ❌   	| ✔️   	| ✔️   	|
| 61  	| DSP: Dense-Sparse Parallel Networks for Self-supervised 3D Multi-person Pose Estimation from Multiple Views                                                                 	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 62  	| DeepHop on Edge: Hop-by-hop Routing byDistributed Learning with Semantic Attention                                                                                          	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ❌   	| ❌   	| ✔️   	| ✔️   	|
| 63  	| Multiverse at the Edge: Interacting Real World and Digital Twins for Wireless Beamforming                                                                                   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ❌   	| ❌   	| ✔️   	| ✔️   	|
| 64  	| A Mobility-Resilient Spectrum Sharing Framework for Operating Wireless UAVs in the 6 GHz Band                                                                               	| ✔️   	| ❔❌  	| ✔️   	| ✔️   	| ❔❌  	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 65  	| DeepBeam: Deep Waveform Learning for Coordination-Free Beam Management in mmWave Networks                                                                                   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ❌   	| ❌   	| ✔️   	| ✔️   	|
| 66  	| An Online Resource Management for Obscured Sensors in Agriculture using UAV                                                                                                 	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 67  	| Whittle Index-Based Q-Learning for Wireless Edge Caching With Linear Function Approximation                                                                                 	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ❌   	| ❌   	| ✔️   	| ✔️   	|
| 68  	| Two-level Computation Offloading to Optimize the Energy Consumption of UAV-mounted Edge Nodes                                                                               	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 69  	| Collaborative WSN-UAV Data Collection in Smart Agriculture: A Bi-objective Optimization Scheme                                                                              	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 70  	| Ultra-Wideband Swarm Ranging Protocol for Dynamic and Dense Networks                                                                                                        	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ❌   	| ❌   	| ✔️   	| ✔️   	|
| 71  	| Fog in the Clouds: UAVs to Provide Edge Computing to IoT Devices                                                                                                            	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 72  	| TRACE: Travel Reinforcement Recommendation Based on Location-Aware Context Extraction                                                                                       	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 73  	| Autonomous UAV Cinematography: A Tutorial and a Formalized Shot-Type Taxonomy                                                                                               	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	|
| 74  	| Task Offloading for Post-Disaster Rescue in Unmanned Aerial Vehicles Networks                                                                                               	| ✔️   	| ❌   	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 75  	| A Unified Programmable Edge Matrix Processor for Deep Neural Networks and Matrix Algebra                                                                                    	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 76  	| MARS: A Multi-Attribute Routing and Scheduling Algorithm for DTN Interplanetary Networks                                                                                    	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ❌   	| ❌   	| ✔️   	| ✔️   	|
| 77  	| Strategic knowledge transfer                                                                                                                                                	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 78  	| Coalition Formation-Based Sub-Channel Allocation in Full-Duplex-Enabled mmWave IABN With D2D                                                                                	| ✔️   	| ❌   	| ❌   	| ✔️   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 79  	| NEMO: Building the Next Generation Meta Operating System                                                                                                                    	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 80  	| GSTAM: Task-Aware Mixture of Grouped Experts with Spatial-Temporal Mamba for Cooperative Multi-Vehicle Pursuit                                                              	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 81  	| Deep Learning Processors for On-Device Intelligence                                                                                                                         	| ✔️   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 82  	| AoI-Aware, Digital Twin-Empowered IoT Query Services in Mobile Edge Computing                                                                                               	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 83  	| Green Industrial Engineering on the Web: Agent-Driven Ant Colony Optimization Tuning for Energy-Efficient 3D Pipe Routing                                                   	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 84  	| Two Hops IRS Optimization in Urban Mobile Environment                                                                                                                       	| ✔️   	| ❌   	| ❌   	| ✔️   	| ❌   	| ❌   	| ❌   	| ✔️   	| ✔️   	|
| 85  	| Assessing the Latent Automated Program Repair Capabilities of Large Language Models using Round-Trip Translation                                                            	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 86  	| A Thread-Level Stream Scheduling Method for Accelerating LVMs’ Inference on a Resource-Constrained Platform                                                                 	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 87  	| The Holy Grail of Multi-Robot Planning: Learning to Generate Online-Scalable Solutions from Offline-Optimal Experts                                                         	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 88  	| Classical Planning in Deep Latent Space                                                                                                                                     	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 89  	| ReduNet: a white-box deep network from the principle of maximizing rate reduction                                                                                           	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 90  	| Exploration on the application of digital twin technology in battlefield environment protection                                                                             	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	|
| 91  	| Bandit Neural Architecture Search for Digital Twin Optimisation: A Scientific Machine Learning Approach                                                                     	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 92  	| DiGrI: Distorted Greedy Approach for Human-Assisted Online Suicide Ideation Detection                                                                                       	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 93  	| An Overview of 3GPP Standardization for Extended Reality (XR) in 5G and Beyond                                                                                              	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	|
| 94  	| Software Engineering for Collective Cyber-Physical Ecosystems                                                                                                               	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	|
| 95  	| Lifting the Fog of Uncertainties: Dynamic Resource Orchestration for the Containerized Cloud                                                                                	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 96  	| Energy- and Quantization-aware DNN Partitioning in the Edge-Cloud Continuum (Work In Progress Paper)                                                                        	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 97  	| A Highly-Efficient Hybrid Simulation System for Flight Controller Design and Evaluation of Unmanned Aerial Vehicles                                                         	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 98  	| Macroprogramming: Concepts, State of the Art, and Opportunities of Macroscopic Behaviour Modelling                                                                          	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	|
| 99  	| Optimal Back-Off Distribution for Maximum Weighted Throughput in CSMA                                                                                                       	| ✔️   	| ❌   	| ❌   	| ✔️   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 100 	| Provably Efficient Resource Allocation for Edge Service Entities Using Hermes                                                                                               	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 101 	| Human-machine collaborative optimization via apprenticeship scheduling                                                                                                      	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 102 	| Technological and Research Challenges in Data Engineering for Sustainable Agriculture                                                                                       	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	|
| 103 	| Non-smooth Newton Methods for Deformable Multi-body Dynamics                                                                                                                	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 104 	| Dancing along Battery: Enabling Transformer with Run-Time Reconfigurability on Mobile Devices                                                                               	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 105 	| Information Retrieval: Advanced Topics and Techniques                                                                                                                       	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	|
| 106 	| CORE-lens: simultaneous communication and object recognition with disentangled-GAN cameras                                                                                  	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 107 	| Challenges for building a cloud native scalable and trustable multi-tenant AIoT platform                                                                                    	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	|
| 108 	| Edge-SLAM: Edge-Assisted Visual Simultaneous Localization and Mapping                                                                                                       	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 109 	| FUDGE: a frugal edge node for advanced IoT solutions in contexts with limited resources                                                                                     	| ✔️   	| ❌   	| ❌   	| ✔️   	| ❌   	| ❌   	| ❌   	| ✔️   	| ✔️   	|
| 110 	| Learned Offline Query Planning via Bayesian Optimization                                                                                                                    	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 111 	| DeepSym: Deep Symbol Generation and Rule Learning for Planning from Unsupervised Robot Interaction                                                                          	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 112 	| A Prediction System Service                                                                                                                                                 	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 113 	| A Primer on Seq2Seq Models for Generative Chatbots                                                                                                                          	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 114 	| PepperPose: Full-Body Pose Estimation with a Companion Robot                                                                                                                	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 115 	| Keep Your Friends Close: Leveraging Affinity Groups to Accelerate AI Inference Workflows                                                                                    	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 116 	| RoS\'{E}: A Hardware-Software Co-Simulation Infrastructure Enabling Pre-Silicon Full-Stack Robotics SoC Evaluation                                                          	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 117 	| MRISA: A Visual Analytics Approach of Locomotion Policies Comparison for Robotics Training                                                                                  	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 118 	| Quartz: time-as-a-service for coordination in geo-distributed systems                                                                                                       	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 119 	| Vertically Autoscaling Monolithic Applications with CaaSPER: Scalable Container-as-a-Service Performance Enhanced Resizing Algorithm for the Cloud                          	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 120 	| Leveraging query logs and machine learning for parametric query optimization                                                                                                	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 121 	| NSGA-Net: neural architecture search using multi-objective genetic algorithm                                                                                                	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 122 	| Autoscaling tiered cloud storage in Anna                                                                                                                                    	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 123 	| Machine Learning with Confidential Computing: A Systematization of Knowledge                                                                                                	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	|
| 124 	| ESTune: Bayesian Uncertainty-Guided Early Stopping for Database Configuration Tuning                                                                                        	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 125 	| Switch transformers: scaling to trillion parameter models with simple and efficient sparsity                                                                                	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 126 	| Lucid: A Non-intrusive, Scalable and Interpretable Scheduler for Deep Learning Training Jobs                                                                                	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 127 	| SODA: An Adaptive Bitrate Controller for Consistent High-Quality Video Streaming                                                                                            	| ✔️   	| ❌   	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ✔️   	|
| 128 	| Socially Aware Path Planning for a Flying Robot in Close Proximity of Humans                                                                                                	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 129 	| From Algorithms to Thinking Machines: The New Digital Power                                                                                                                 	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	|
| 130 	| Bridging the semantic gap between qualitative and quantitative models of distributed systems                                                                                	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 131 	| Beyond Life and Death: Exploring Digital Legacy with Spatial Media, Emerging Technologies, and Evolving Ethics                                                              	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	|
| 132 	| Weaving Fire into Form: Aspirations for Tangible and Embodied Interaction                                                                                                   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	|
| 133 	| Enhancing Taxonomy Completion with Concept Generation via Fusing Relational Representations                                                                                 	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	|
| 134 	| Technical Report Column                                                                                                                                                     	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	|
| 135 	| Improving Performance-Power-Programmability in Space Avionics with Edge Devices: VBN on Myriad2 SoC                                                                         	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 136 	| Tackling Climate Change with Machine Learning                                                                                                                               	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	|
| 137 	| Intersection-free rigid body dynamics                                                                                                                                       	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 138 	| Seminal Graphics Papers: Pushing the Boundaries, Volume 2                                                                                                                   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	|
| 139 	| Probabilistic and Causal Inference: The Works of Judea Pearl                                                                                                                	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	|
| 140 	| Multi-robot collaborative dense scene reconstruction                                                                                                                        	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 141 	| The Handbook of Multimodal-Multisensor Interfaces: Language Processing, Software, Commercialization, and Emerging Directions                                                	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	|
| 142 	| Sparse additive Gaussian process regression                                                                                                                                 	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 143 	| **Learning to Fly: A Distributed Deep Reinforcement Learning Framework for Software-Defined UAV Network Control**                                                           	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 144 	| Multi-Hop Data Offloading in Heterogeneous Ad-Hoc UAV Networks with Deep Reinforcement Learning                                                                             	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	|
| 145 	| MPRdeep: Multi-Objective Joint Optimal Node Positioning and Resource Allocation for FANETs with Deep Reinforcement learning                                                 	| ✔️   	| ✔️   	| ✔️   	| ❔❌  	| ✔️   	| ❔❌  	| ✔️   	| ✔️   	| ✔️   	|
| 146 	| **DRL-Based Dynamic Multi-Modal Network Handover for Low-Altitude UAVs**                                                                                                    	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 147 	| Blockchain-Enabled Multi-Agent Deep Reinforcement Learning for Task Offloading in UAV-Assisted Vehicular Edge Computing                                                     	| ✔️   	| ✔️   	| ❌   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ❌   	|
| 148 	| Tailoring FANET-Based 6G Network Slices in Remote Areas for Low-Latency Applications                                                                                        	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ❌   	| ❌   	| ✔️   	| ✔️   	|
| 149 	| **Energy-Efficient Narwhal Optimization-Based Cooperative Routing Protocol for Stable FANETs (NOA-CRP-SFANET)**                                                             	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 150 	| Secure Military Surveillance Using AI-Enabled Swarm Drones and Blockchain-Backed IoT Networks                                                                               	| ✔️   	| ✔️   	| ❌   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ❌   	|
| 151 	| Reinforcement Learning for Resource Planning in Drone-Based Softwarized Networks                                                                                            	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ❌   	|
| 152 	| Optimizing Dynamic Deployment of UAV Base Stations: A Digital Twin Approach                                                                                                 	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ❔❌  	|
| 153 	| **Reinforcement Learning-based Scheduling for UAV-assisted Ad-hoc Computing-aware Network**                                                                                 	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 154 	| **Edge-Assisted Intelligent Routing Framework for Reliable Data Collection in UAV Networks Using Reinforcement Learning**                                                   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 155 	| **Blockchain-Based Distributed Resource Management for Delay Minimization in UAV-Assisted Vehicular Networks**                                                              	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 156 	| **AI-Enabled Cross-Layer QoS Routing Framework for Mission-Critical 5G/6G-Integrated MANETs and UAV Swarms**                                                                	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 157 	| ROBIN: A Distributed MARL Approach for Dynamic Bandwidth Optimization in Drone Ad-Hoc Networks                                                                              	| ✔️   	| ❔❌  	| ✔️   	| ✔️   	| ❔❌  	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 158 	| Reinforcement Learning Based Energy-Efficient Routing with Latency Constraints for FANETs                                                                                   	| ✔️   	| ❔❌  	| ✔️   	| ✔️   	| ❔❌  	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 159 	| Rapid Deployment and QoS Provisioning in Coverage Based Multiple UAV Enabled Emergency Communication Networks                                                               	| ✔️   	| ❔❌  	| ✔️   	| ✔️   	| ❔❌  	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 160 	| A Novel UAV-Assisted VANET Routing Protocol for Post-Disaster Emergency Communications                                                                                      	| ✔️   	| ❔❌  	| ✔️   	| ✔️   	| ❔❌  	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 161 	| Deep Learning Empowered Service Selection in FANETs: A 5G-Enabled UAV-Cloud Architecture                                                                                    	| ✔️   	| ❌   	| ❌   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ❌   	|
| 162 	| ICRA: An Intelligent Clustering Routing Approach for UAV Ad Hoc Networks                                                                                                    	| ✔️   	| ❔❌  	| ✔️   	| ✔️   	| ❔❌  	| ✔️   	| ✔️   	| ✔️   	| ❔✔️  	|
| 163 	| Novel Robust Dynamic Distributed Drone-Deployment Strategy for Channel-Capacity Optimization for 3-D UAV-Aided Ad Hoc Networks                                              	| ✔️   	| ❌   	| ✔️   	| ❌   	| ❌   	| ❌   	| ✔️   	| ✔️   	| ✔️   	|
| 164 	| STAR-RIS Based Resource Scheduling and Mode Selection for Drone Assisted 5G Communications                                                                                  	| ✔️   	| ❔❌  	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ❔❌  	| ✔️   	| ✔️   	|
| 165 	| Machine Learning to Enhance the V2V Communication-based IoD Using Companding Dataset                                                                                        	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ❔❌  	| ✔️   	| ✔️   	|
| 166 	| Skeleton-Based Swarm Routing (SSR): Intelligent Smooth Routing for Dynamic UAV Networks                                                                                     	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 167 	| Optimal 3D Placement for Integrated Access Backhauling in UAV-Assisted Wireless Networks Using Reinforcement Learning                                                       	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ❌   	| ❌   	| ✔️   	| ✔️   	|
| 168 	| **Enhancing Routing Performance Through Trajectory Planning With DRL in UAV-Aided VANETs**                                                                                  	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 169 	| **Design of a Priority-Differentiated MAC Protocolfor UANETs Based on MAD3QN**                                                                                              	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 170 	| Three-Dimensional Mobility Management of Unmanned Aerial Vehicles in Flying Ad-Hoc Networks                                                                                 	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ❌   	| ✔️   	| ✔️   	| ✔️   	|
| 171 	| **Reinforcement Learning Based Energy-Efficient Fast Routing for FANETs**                                                                                                   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 172 	| **UAV Assisted Cooperative Caching on Network Edge Using Multi-Agent Actor-Critic Reinforcement Learning**                                                                  	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 173 	| **Multiagent Deep Reinforcement Learning-Based Hierarchical Scheduling in Heterogeneous UAV-Enabled Vehicular Networks**                                                    	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 174 	| MA-QoS ICN: Mobility-aware QoS ICN framework with Independent Q-Learning for UAVs in Disaster Scenarios                                                                     	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 175 	| Drone Swarms for Post-Disaster Search and Rescue in Remote and Inaccessible Areas                                                                                           	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	|
| 176 	| Cluster-Based Machine Learning-Driven Routing for UAV Networks in 6G Environment                                                                                            	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 177 	| **Adaptive Routing in MANETs using Hydro-AI: A Hybrid Deep Reinforcement Learning-based Approach**                                                                          	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 178 	| Dual-Mode Energy Harvesting-Based Aerial Computing in NOMA-IoV Networks via Deep Reinforcement Learning                                                                     	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ❌   	|
| 179 	| Latency-Aware Load Distribution Algorithm for Microservice Deployment in UAV Networks                                                                                       	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 180 	| **Energy-efficient RL-based aerial network deployment testbed for disaster areas**                                                                                          	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 181 	| A Reinforcement Learning Based Service Scheduling Algorithm for Internet of Drones                                                                                          	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 182 	| **Priority-Aware Task Offloading in UAV-Assisted Satellite MEC Networks**                                                                                                   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 183 	| UAV Assisted Multi-Attack Detection Method for Vehicular Ad Hoc Networks                                                                                                    	| ✔️   	| ✔️   	| ❌   	| ❌   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	|
| 184 	| Intelligent Foundation Model-Enabled Low-Altitude FANETs: Integrating Communication, Control, Sensing, and Security                                                         	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	|
| 185 	| Particle Swarm Optimization-Based Content Delivery Protocol for UAV VAENTs                                                                                                  	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 186 	| Optimizing Energy Efficiency and Latency in UAV-Assisted 6G Networks through Intelligent Routing Algorithms - A Systematic Review                                           	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	|
| 187 	| **Lyapunov-Guided Deep Reinforcement Learning for Task Offloading in Multi-UAV-Assisted Edge Computing Networks**                                                           	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 188 	| Intelligent Air-Ground Cooperative Relay Assisted Secure mmWave Vehicular Communications                                                                                    	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ❌   	|
| 189 	| **Spatiotemporal-Aware Resilient Routing Algorithm Based on Graph Attention Double DQN**                                                                                    	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 190 	| **GULL: Graph-guided Hierarchical Reinforcement Learning for Intelligent Aerial Ferry Routing in Digital Twins**                                                            	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 191 	| Improved Q-Learning-Based Multi-Hop Routing for UAV-Assisted Communication                                                                                                  	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 192 	| Enhancing Vehicular Network Performance through Integrated RSU and UAV Deployment                                                                                           	| ✔️   	| ❌   	| ✔️   	| ❌   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 193 	| Joint UAV Trajectory Scheduling and Network Routing for FANETs: A Reinforcement Learning Approach                                                                           	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 194 	| AI-Enabled Vehicular Data Offloading for Sustainable Smart Cities: Taxonomy, KPI Models, and Open Challenges                                                                	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	|
| 195 	| A Framework to Develop and Validate RL-Based Obstacle-Aware UAV Positioning Algorithms                                                                                      	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 196 	| Enabling Intelligent IoCV Services at the Edge for 5G Networks and Beyond                                                                                                   	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	|
| 197 	| **Multi-Agent Meta Reinforcement Learning for Reliable and Low-Latency Distributed Inference in Resource-Constrained UAV Swarms**                                           	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 198 	| **Learning Based Rate Adapter for UAV Streaming**                                                                                                                           	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 199 	| Design and Analysis of Swarm Behaviours Using Centralized and Decentralized Communication Protocols                                                                         	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	|
| 200 	| A Learning Automaton MAC Protocol for Directional FANETs With Throughput Enhancement and Fairness Control                                                                   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 201 	| Deep Learning-Driven Resource Allocation for MEC-Enabled UAV Collision Avoidance System                                                                                     	| ✔️   	| ❌   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ✔️   	|
| 202 	| Robust Peer-to-Peer Selection in FANETs With Uncertain Link Quality: A SAO-Based Approach for Improving Throughput, Fairness, and Energy Efficiency                         	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 203 	| Design of a 5G Network Slice Extension With MEC UAVs Managed With Reinforcement Learning                                                                                    	| ✔️   	| ❌   	| ❌   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ❌   	|
| 204 	| Multi-Tier Variable Height UAV Networks: User Coverage and Throughput Optimization                                                                                          	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 205 	| Enhancement in 6G Intelligent Network Technologies using Cognitive Radio Network                                                                                            	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ❌   	| ❌   	| ✔️   	| ✔️   	|
| 206 	| AI and Ml based systemic analysis on Highly Advanced Life Threatening Cybernetics for Future Wireless Network                                                               	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	|
| 207 	| Secure Airspace Based on Virtual Trust Bubbles: Blockchain-Assisted V2V Negotiation and Quantum Optimization for Collision-Free UAV Coordination in Zero-Trust Environments 	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 208 	| Co-Operative Caching for Real-Time Content Retrieval in an Integrated Space-Air-Ground Network for a Post-Disaster Scenario                                                 	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 209 	| Hierarchical Aggregation and Cooperative Caching for Decentralized Federated Learning in UAV-Assisted Internet of Vehicles                                                  	| ✔️   	| ✔️   	| ❌   	| ❌   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	|
| 210 	| UAV-Assisted Internet of Vehicles Over Licensed and Unlicensed Spectrum: Architecture, Intelligent Resource Management, and Challenges                                      	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	|
| 211 	| Advances in Machine Learning-Driven Cognitive Radio for Wireless Networks: A Survey                                                                                         	| ✔️   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	|
| 212 	| **Task Offloading in Hierarchical UAV-Enabled MEC Network: A Trust Region Policy Optimization-Based Approach**                                                              	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 213 	| An Adaptive Q-Value Adjustment-Based Learning Model for Reliable Vehicle-to-UAV Computation Offloading                                                                      	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ❌   	|
| 214 	| Energy-Based Resource Allocation in UAV-Assisted Mobile Networks for Public Safety                                                                                          	| ✔️   	| ❔❌  	| ✔️   	| ✔️   	| ❔❌  	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 215 	| A Digital Twin Framework for Delay-Doppler Enabled Multi-UAV Selection Using Neural Contextual Bandits                                                                      	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ❌   	| ✔️   	| ✔️   	|
| 216 	| Disaster-Resilient Emergency Communication With Intelligent Air–Ground Cooperation                                                                                          	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 217 	| Multi-UAV Cooperative Task Offloading and Resource Allocation in 5G Advanced and Beyond                                                                                     	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 218 	| Blockchain-Enabled Task Offloading and Trajectory Optimization for UAV-Assisted Vehicular Edge Computing                                                                    	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ❌   	|
| 219 	| **QL-AODV: Q-Learning-Enhanced Multi-Path Routing Protocol for 6G-Enabled Autonomous Aerial Vehicle Networks**                                                              	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 220 	| Joint Resource Allocation for V2X Communications with Multi-Type Mean-Field Reinforcement Learning                                                                          	| ✔️   	| ❌   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ❌   	| ✔️   	| ❌   	|
| 221 	| ~~Energy-Efficient Narwhal Optimization-based Cooperative Routing Protocol for Stable FANETs (NOA-CRP-SFANET)~~                                                             	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	|
| 222 	| Q-FANETGS-BS: A Six-State Routing Model in FANET for Performing Efficient Data Transfer                                                                                     	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 223 	| **Joint Resource Allocation for UAV-Assisted V2X Communication With Mean Field Multi-Agent Reinforcement Learning**                                                         	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 224 	| DroneCOCoNet: Learning-based edge computation offloading and control networking for drone video analytics                                                                   	| ✔️   	| ❔❌  	| ✔️   	| ✔️   	| ❔❌  	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 225 	| **MPRdeep: Multi-objective joint optimal node positioning and resource allocation for FANETs with deep reinforcement learning**                                             	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 226 	| A Multi-Agent Federated DRL Model for Vehicular Task Offloading in WPT-Aided eROAD Environment                                                                              	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ✔️   	|
| 227 	| Diffusion-based Reinforcement Learning for Dynamic UAV-assisted Vehicle Twins Migration in Vehicular Metaverses                                                             	| ✔️   	| ✔️   	| ✔️   	| ❌   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ❌   	|
| 228 	| Monitoring System-Based Flying IoT in Public Health and Sports Using Ant-Enabled Energy-Aware Routing                                                                       	| ✔️   	| ❌   	| ❌   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 229 	| A novel Q-learning-based secure routing scheme with a robust defensive system against wormhole attacks in flying ad hoc networks                                            	| ✔️   	| ❌   	| ❌   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 230 	| An Improved Reinforcement Learning-Based 6G UAV Communication for Smart Cities                                                                                              	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 231 	| Relay Selection for UAV-Assisted Urban Vehicular Ad Hoc Networks                                                                                                            	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ❌   	| ✔️   	| ✔️   	|
| 232 	| **Resource Allocation and Trajectory Optimization in Multi-UAV Collaborative Vehicular Networks: An Extended Multiagent DRL Approach**                                      	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 233 	| Optimizing UAV-assisted IoT sensor networks: A multi-objective approach to data collection and routing                                                                      	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 234 	| **DRL-Based Multidimensional Resource Scheduling for Intelligent Connected Vehicles in UAV-Assisted VEC Systems**                                                           	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 235 	| Digital Twin Empowered mmWave Multi-Hop V2X Routing Scheme With UAV Assistance                                                                                              	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 236 	| **Deep reinforcement learning enhanced skeleton based pipe routing for high-throughput transmission in flying ad-hoc networks**                                             	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 237 	| **FD3QN: A Federated Deep Reinforcement Learning Approach for Cross-Domain Resource Cooperative Scheduling in Hybrid Cloud Architecture**                                   	| ❔✔️  	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 238 	| Multi-agent Deep Learning for QoS-aware Task Scheduling in UAV-assisted Wireless Ad Hoc Networks                                                                            	| ❔❌  	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 239 	| Artificial Intelligence-Enabled Fully Echoed Q-Routing and Adaptive Directional Medium Access Control Protocol for Flying Ad-Hoc Networks                                   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ❔❌  	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 240 	| Cost-Efficient Deployment Optimization for Multi-UAV-Assisted Vehicular Edge Computing Networks                                                                             	| ✔️   	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	|
| 241 	| Information Loss Minimization in FANET-Aided Rechargeable IoT Networks Using Q-Learning                                                                                     	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 242 	| Slicing a FANET for heterogeneous delay-constrained applications                                                                                                            	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ❌   	|
| 243 	| Delay aware downlink resource allocation scheme for future generation tactical wireless networks                                                                            	| ❔❌  	| ❌   	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 244 	| Green Edge Intelligence for Smart Management of a FANET in Disaster-Recovery Scenarios                                                                                      	| ✔️   	| ❔❌  	| ❌   	| ✔️   	| ❔❌  	| ✔️   	| ✔️   	| ✔️   	| ❔❌  	|
| 245 	| UAV-Assisted Heterogeneous Multi-Server Computation Offloading With Enhanced Deep Reinforcement Learning in Vehicular Networks                                              	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ❔❌  	| ✔️   	| ❔❌  	|
| 246 	| **Quantum-secured routing in drone communication for 6G-enabled smart mobility**                                                                                            	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 247 	| AI-Driven UAV-Assisted Edge Computing for Rapid Response in Emergency Wireless Networks                                                                                     	| ❔❌  	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 248 	| **Learning-Based Energy-Efficient Anti-Jamming FANET Routing With QoS Guarantee**                                                                                           	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 249 	| ~~_Energy-Efficient RL-Based Aerial Network Deployment Testbed for Disaster Areas_~~                                                                                        	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	| ❌   	|
| 250 	| DQ-MAC: AN ADAPTIVE MAC PROTOCOL SWITCHING SCHEME BASED ON DEEP Q-LEARNING                                                                                                  	| ❔❌  	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 251 	| Reinforcement-Learning-Based Geographic Routing Considering Future Evolution of Link States for UAV Networks                                                                	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 252 	| **MDR:Multilayer Perceptron-Based Deep Reinforcement Learning FANET Routing Approach**                                                                                      	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 253 	| QEHLR: A Q-Learning Empowered Highly Dynamic and Latency-Aware Routing Algorithm for Flying Ad-Hoc Networks                                                                 	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 254 	| Reinforcement Learning Based Anti-Jamming FANET Routing with QoS Guarantee                                                                                                  	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 255 	| Q-Learning Based Multi-objective Optimization Routing Strategy in UAVs Deterministic Network                                                                                	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 256 	| Q-LBR: Q-learning based load balancing routing for UAV-assisted VANET                                                                                                       	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 257 	| **Multi-Agent Deep Reinforcement Learning in Flying Ad-Hoc Networks for Delay-Constrained Applications**                                                                    	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 258 	| A New Covid-19 Tracing Approach using Machine Learning and Drones Enabled Wireless Network                                                                                  	| ✔️   	| ✔️   	| ❌   	| ❌   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	|
| 259 	| FS-MAC: An Adaptive MAC Protocol with Fault-Tolerant Synchronous Switching for FANETs                                                                                       	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 260 	| **Deep reinforcement learning-based routing framework for bidirectional communication in UAV-UGV networks**                                                                 	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 261 	| Spatiotemporal Precise Routing Strategy for Multi-UAV-Based Power Line Inspection Using Hybrid Network of FANET and Satellite Internet                                      	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	|
| 262 	| Transmission Control and Optimization in Next-generation Hybrid Wireless Networks: An Online Reinforcement Learning Approach                                                	| ❔❌  	| ❌   	| ✔️   	| ✔️   	| ❌   	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|
| 287 	| H-HOME: A learning framework of federated FANETs to provide edge computing to future delay-constrained IoT systems                                                          	| ✔️   	| ❔✔️  	| ✔️   	| ✔️   	| ❔✔️  	| ✔️   	| ✔️   	| ✔️   	| ✔️   	|

In [34]:
idx_to_keep: list[int] = [
    2, 6, 11, 33, 143, 146,
    149, 153, 154, 155, 156, 168,
    169, 171, 172, 173, 177, 180,
    182, 187, 189, 190, 197, 198,
    212, 219, 223, 225, 232, 234,
    236, 237, 246, 248, 252, 257, 
    260 
]
adjusted_idx = [idx-1 for idx in idx_to_keep] # Adjusting for zero-based indexing
slr_df_final = slr_df_clean.loc[adjusted_idx]
print(f'No. of entries in the final SLR DataFrame: {slr_df_final.shape[0]}')
print(slr_df_final[['title']])

No. of entries in the final SLR DataFrame: 37
                                                 title
1    Topology Maintenance Optimization Algorithm Ba...
5    Distributed trained Q-learning based AODV rout...
10   Deep Distributional Reinforcement Learning-Bas...
32   Learning Driven Mobility Control of Airborne B...
142  Learning to Fly: A Distributed Deep Reinforcem...
145  DRL-Based Dynamic Multi-Modal Network Handover...
148  Energy-Efficient Narwhal Optimization-Based Co...
152  Reinforcement Learning-based Scheduling for UA...
153  Edge-Assisted Intelligent Routing Framework fo...
154  Blockchain-Based Distributed Resource Manageme...
155  AI-Enabled Cross-Layer QoS Routing Framework f...
167  Enhancing Routing Performance Through Trajecto...
168  Design of a Priority-Differentiated MAC Protoc...
170  Reinforcement Learning Based Energy-Efficient ...
171  UAV Assisted Cooperative Caching on Network Ed...
172  Multiagent Deep Reinforcement Learning-Based H...
176  Adaptive Routi

### Step 4.3: Full text screening

| index 	| title                                                                                                                                 	| IC1 	| IC2 	| IC3 	| IC4 	| EC1 	| EC2 	| EC3 	| EC4 	| EC5 	|
|-------	|---------------------------------------------------------------------------------------------------------------------------------------	|-----	|-----	|-----	|-----	|-----	|-----	|-----	|-----	|-----	|
| 2     	| Topology Maintenance Optimization Algorithm Based on Deep Reinforcement Learning in High Dynamic Flying Ad-Hoc Networks               	|  ✔️  	|   ✔️ 	|   ✔️ 	|   ✔️ 	|   ✔️	|  ✔️  	|  ✔️  	|  ✔️  	|  ✔️  	|
| 6     	| Distributed trained Q-learning based AODV routing protocol for UAV ad hoc networks                                                    	|  ✔️  	|  ❌   |   ✔️  |   ✔️  |   ❌  |  ✔️   |  ✔️   |  ✔️   |  ✔️   |
| 11    	| Deep Distributional Reinforcement Learning-Based Adaptive Routing With Guaranteed Delay Bounds                                        	|  ✔️   |  ✔️  	|   ✔️  |   ✔️  |   ✔️  |  ✔️  	|   ❌  |  ✔️   |  ✔️   |
| 33    	| **Learning Driven Mobility Control of Airborne Base Stations in Emergency Networks**                                                    	|  ✔️ 	|  ✔️  	|   ✔️ 	|   ✔️ 	|   ✔️ 	|  ✔️  	|  ✔️ 	|  ✔️  	|  ✔️  	|
| 143   	| **Learning to Fly: A Distributed Deep Reinforcement Learning Framework for Software-Defined UAV Network Control**                       	|  ✔️ 	|  ✔️  	|   ✔️ 	|   ✔️ 	|   ✔️ 	|  ✔️  	|  ✔️ 	|  ✔️  	|  ✔️  	|
| 146   	| DRL-Based Dynamic Multi-Modal Network Handover for Low-Altitude UAVs                                                                  	|  ✔️   |  ✔️  	|   ✔️  |   ✔️  |   ✔️  |  ✔️  	|   ❌  |  ✔️   |  ✔️   |
| 149   	| Energy-Efficient Narwhal Optimization-Based Cooperative Routing Protocol for Stable FANETs (NOA-CRP-SFANET)                           	|  ✔️  	|  ❌   |   ✔️  |   ✔️  |   ❌  |  ✔️   |  ✔️   |  ✔️   |  ✔️   |
| 153   	| Reinforcement Learning-based Scheduling for UAV-assisted Ad-hoc Computing-aware Network                                               	|  ✔️   |  ✔️  	|   ✔️  |   ✔️  |   ✔️  |  ✔️  	|   ❌  |  ✔️   |  ✔️   |
| 154   	| **Edge-Assisted Intelligent Routing Framework for Reliable Data Collection in UAV Networks Using Reinforcement Learning**               	|  ✔️ 	|  ✔️  	|   ✔️ 	|   ✔️ 	|   ✔️ 	|  ✔️  	|  ✔️ 	|  ✔️  	|  ✔️  	|
| 155   	| **Blockchain-Based Distributed Resource Management for Delay Minimization in UAV-Assisted Vehicular Networks**                           	|  ✔️ 	|  ✔️  	|   ✔️ 	|   ✔️ 	|   ✔️ 	|  ✔️  	|  ✔️ 	|  ✔️  	|  ✔️  	|
| 156   	| AI-Enabled Cross-Layer QoS Routing Framework for Mission-Critical 5G/6G-Integrated MANETs and UAV Swarms                              	|  ✔️  	|  ❌   |   ✔️  |   ✔️  |   ❌  |  ✔️   |  ✔️   |  ✔️   |  ✔️   |
| 168   	| **Enhancing Routing Performance Through Trajectory Planning With DRL in UAV-Aided VANETs**                                              	|  ✔️ 	|  ✔️  	|   ✔️ 	|   ✔️ 	|   ✔️ 	|  ✔️  	|  ✔️ 	|  ✔️  	|  ✔️  	|
| 169   	| Design of a Priority-Differentiated MAC Protocolfor UANETs Based on MAD3QN                                                            	|     	|     	|     	|     	|     	|     	|     	|     	|     	|
| 171   	| Reinforcement Learning Based Energy-Efficient Fast Routing for FANETs                                                                 	|     	|     	|     	|     	|     	|     	|     	|     	|     	|
| 172   	| UAV Assisted Cooperative Caching on Network Edge Using Multi-Agent Actor-Critic Reinforcement Learning                                	|     	|     	|     	|     	|     	|     	|     	|     	|     	|
| 173   	| Multiagent Deep Reinforcement Learning-Based Hierarchical Scheduling in Heterogeneous UAV-Enabled Vehicular Networks                  	|     	|     	|     	|     	|     	|     	|     	|     	|     	|
| 177   	| Adaptive Routing in MANETs using Hydro-AI: A Hybrid Deep Reinforcement Learning-based Approach                                        	|     	|     	|     	|     	|     	|     	|     	|     	|     	|
| 180   	| Energy-efficient RL-based aerial network deployment testbed for disaster areas                                                        	|     	|     	|     	|     	|     	|     	|     	|     	|     	|
| 182   	| Priority-Aware Task Offloading in UAV-Assisted Satellite MEC Networks                                                                 	|     	|     	|     	|     	|     	|     	|     	|     	|     	|
| 187   	| Lyapunov-Guided Deep Reinforcement Learning for Task Offloading in Multi-UAV-Assisted Edge Computing Networks                         	|     	|     	|     	|     	|     	|     	|     	|     	|     	|
| 189   	| Spatiotemporal-Aware Resilient Routing Algorithm Based on Graph Attention Double DQN                                                  	|     	|     	|     	|     	|     	|     	|     	|     	|     	|
| 190   	| GULL: Graph-guided Hierarchical Reinforcement Learning for Intelligent Aerial Ferry Routing in Digital Twins                          	|     	|     	|     	|     	|     	|     	|     	|     	|     	|
| 197   	| Multi-Agent Meta Reinforcement Learning for Reliable and Low-Latency Distributed Inference in Resource-Constrained UAV Swarms         	|     	|     	|     	|     	|     	|     	|     	|     	|     	|
| 198   	| Learning Based Rate Adapter for UAV Streaming                                                                                         	|     	|     	|     	|     	|     	|     	|     	|     	|     	|
| 212   	| Task Offloading in Hierarchical UAV-Enabled MEC Network: A Trust Region Policy Optimization-Based Approach                            	|     	|     	|     	|     	|     	|     	|     	|     	|     	|
| 219   	| QL-AODV: Q-Learning-Enhanced Multi-Path Routing Protocol for 6G-Enabled Autonomous Aerial Vehicle Networks                            	|     	|     	|     	|     	|     	|     	|     	|     	|     	|
| 223   	| Joint Resource Allocation for UAV-Assisted V2X Communication With Mean Field Multi-Agent Reinforcement Learning                       	|     	|     	|     	|     	|     	|     	|     	|     	|     	|
| 225   	| MPRdeep: Multi-objective joint optimal node positioning and resource allocation for FANETs with deep reinforcement learning           	|     	|     	|     	|     	|     	|     	|     	|     	|     	|
| 232   	| Resource Allocation and Trajectory Optimization in Multi-UAV Collaborative Vehicular Networks: An Extended Multiagent DRL Approach    	|     	|     	|     	|     	|     	|     	|     	|     	|     	|
| 234   	| DRL-Based Multidimensional Resource Scheduling for Intelligent Connected Vehicles in UAV-Assisted VEC Systems                         	|     	|     	|     	|     	|     	|     	|     	|     	|     	|
| 236   	| Deep reinforcement learning enhanced skeleton based pipe routing for high-throughput transmission in flying ad-hoc networks           	|     	|     	|     	|     	|     	|     	|     	|     	|     	|
| 237   	| FD3QN: A Federated Deep Reinforcement Learning Approach for Cross-Domain Resource Cooperative Scheduling in Hybrid Cloud Architecture 	|     	|     	|     	|     	|     	|     	|     	|     	|     	|
| 246   	| Quantum-secured routing in drone communication for 6G-enabled smart mobility                                                          	|     	|     	|     	|     	|     	|     	|     	|     	|     	|
| 248   	| Learning-Based Energy-Efficient Anti-Jamming FANET Routing With QoS Guarantee                                                         	|     	|     	|     	|     	|     	|     	|     	|     	|     	|
| 252   	| MDR:Multilayer Perceptron-Based Deep Reinforcement Learning FANET Routing Approach                                                    	|     	|     	|     	|     	|     	|     	|     	|     	|     	|
| 257   	| Multi-Agent Deep Reinforcement Learning in Flying Ad-Hoc Networks for Delay-Constrained Applications                                  	|     	|     	|     	|     	|     	|     	|     	|     	|     	|
| 260   	| Deep reinforcement learning-based routing framework for bidirectional communication in UAV-UGV networks                               	|     	|     	|     	|     	|     	|     	|     	|     	|     	|

In [35]:
slr_df_final.insert(len(slr_df_final.columns), 'IC1', '')
slr_df_final.insert(len(slr_df_final.columns), 'IC2', '')
slr_df_final.insert(len(slr_df_final.columns), 'IC3', '')
slr_df_final.insert(len(slr_df_final.columns), 'IC4', '')
slr_df_final.insert(len(slr_df_final.columns), 'EC1', '')
slr_df_final.insert(len(slr_df_final.columns), 'EC2', '')
slr_df_final.insert(len(slr_df_final.columns), 'EC3', '')
slr_df_final.insert(len(slr_df_final.columns), 'EC4', '')
slr_df_final.insert(len(slr_df_final.columns), 'EC5', '')
slr_df_final.insert(0, 'index', idx_to_keep)
slr_df_final.set_index('index', inplace=True)
slr_df_final[['title', 'IC1', 'IC2', 'IC3', 'IC4', 'EC1', 'EC2', 'EC3', 'EC4', 'EC5']].to_csv('slr_data_full_text_screening.csv')

## Step ?: Quality assestment
The following dimensions are going to be evaluated:

1. (QA1) Is the problem clearly described and well formulated?

2. (QA2) Is data collection or generation (through simulation) explained?

3. (QA3) Is model design and training explained?

4. (QA4) Are implementation details provided?

5. (QA4) Are results clearly presented in terms of networking metrics?

6. (QA5) Are limitations/generalization hability discussed?


With these questions I aim to evaluate: research design (QA1, QA2 and QA3), reproducibility (QA2, QA3 and QA4) and relevance (QA4 and QA5)

## Step 5: Extract data from studies resulting from step 4.
To homogenise data extraction, the following template is going to be used:

